File: src/test/java/at/yawk/fiction/impl/fanfiction/ChapterPageParserTest.java
package at.yawk.fiction.impl.fanfiction;

import at.yawk.fiction.impl.ParseTest;
import org.testng.annotations.Test;

/**
 * @author yawkat
 */
public class ChapterPageParserTest {
    @Test
    public void test() throws Exception {
        ParseTest.print(ParseTest.parse(ChapterPageParser.class,
                                        "https://www.fanfiction.net/s/11106521/1").getStory());
    }
}

File: src/test/kotlin/at/yawk/fiction/impl/fanfiction/ChapterPageParserTest.kt
package at.yawk.fiction.impl.fanfiction

import at.yawk.fiction.impl.ParseTest
import org.testng.annotations.Test

/**
 * @author yawkat
 */
class ChapterPageParserTest {
    @Test
    fun test() {
        ParseTest.print(ParseTest.parse(ChapterPageParser, "https://www.fanfiction.net/s/7031899/1").first)
    }
}